drivers/phy/qualcomm/phy-qcom-qusb2.c

```diff
@@ -810,6 +851,10 @@ static const struct phy_ops qusb2_phy_gen_ops = {
 
 static const struct of_device_id qusb2_phy_of_match_table[] = {
 	{
+
```

Comment:
Redundant empty line.

Reply:
I think it is OK now + checked.
Please verify

Reply:
Looks good to me.